### PR TITLE
[libterminal, terminal_view] replace std::bind with lambdas

### PR DESCRIPTION
**Pull request by 3n16m4**
- update README (I also seriously need to learn git)

### CODE REVIEW

File: src/terminal_view/Renderer.cpp [hunk 1]
```diff
@@ -161,7 +161,7 @@ uint64_t Renderer::render(Terminal& _terminal,
             }
 
             changes = _terminal.preRender(_now);
-            _terminal.screen().render([this](cursor_pos_t _row, cursor_pos_t _col, Cell const& _cell) { renderCell(_row, _col, _cell); },
+            _terminal.screen().render([this](cursor_pos_t _row, cursor_pos_t _col, terminal::ScreenBuffer::Cell const& _cell) { renderCell(_row, _col, _cell); },
```
Comment (christianparpart) [inline src/terminal_view/Renderer.cpp:164]:
```suggestion
            _terminal.screen().render([this](cursor_pos_t _row, cursor_pos_t _col, ScreenBuffer::Cell const& _cell) { renderCell(_row, _col, _cell); },
```

File: src/terminal_view/Renderer.cpp [hunk 2]
```diff
@@ -170,7 +170,7 @@ uint64_t Renderer::render(Terminal& _terminal,
         else
         {
             changes = _terminal.preRender(_now);
-            _terminal.screen().render([this](cursor_pos_t _row, cursor_pos_t _col, Cell const& _cell) { renderCell(_row, _col, _cell); },
+            _terminal.screen().render([this](cursor_pos_t _row, cursor_pos_t _col, terminal::ScreenBuffer::Cell const& _cell) { renderCell(_row, _col, _cell); },
```
Comment (christianparpart) [inline src/terminal_view/Renderer.cpp:173]:
```suggestion
            _terminal.screen().render([this](cursor_pos_t _row, cursor_pos_t _col, ScreenBuffer::Cell const& _cell) { renderCell(_row, _col, _cell); },
```

Comment (3n16m4) [reply src/terminal_view/Renderer.cpp:173]: that looks right. it builds fine without the namespace 'terminal'.

**Comment by christianparpart** [comment]
Do you need support in getting rid of the merge conflict?

**Comment by 3n16m4** [comment]
i fixed the merge conflict, should be good now!

**Pull request merged**